### DISCUSSION TITLE
18.0.0

### DISCUSSION THREAD

**Discussion by jamesdaniels** [angular/angularfire#3535] [General]
## What's Changed
* Support Angular 18 peers by @jamesdaniels in https://github.com/angular/angularfire/pull/3533 and https://github.com/angular/angularfire/pull/3534

**Full Changelog**: https://github.com/angular/angularfire/compare/17.1.0...18.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/angular/angularfire/releases/tag/18.0.0'>18.0.0</a>.</em>

**Comment by davidbusuttil** [comment]
Is there any reason why @angular/fire is not being updated with the rest of @angular libraries?

**Comment by yihubertdev** [comment]
I'm wondering zoneless version of the angular fire release?
